index.html

```diff
@@ -115,6 +115,33 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+         <!-- ________ amyyalex card START ________  -->
+         <div class="card">
+          <p class="name">amyyalex</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/AlexStephanie10" target="_blank">@AlexStephanie10</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/amyyalex" target="_blank">Github</a>
```

Comment:
```suggestion
            <a href="https://github.com/amyyalex" target="_blank">@amyyalex</a>
```

Reply:
I just corrected it, thanks for pointing that out